ios/FluentUI/Tab Bar/TabBarItem.swift

```diff
@@ -33,60 +32,55 @@ open class TabBarItem: NSObject {
         }
     }
 
-    /// Initializes `TabBarItem`
-    /// - Parameter title: Used for tabbar item view's label and for its accessibilityLabel.
+    /// Initializes `TabBarItem`             /// Notification sent when the tab bar item's badge value changes.
```

Comment:
Not sure if it's a missing line break here or it's just just the preview that is looking weird.